src/features/security-cloud-game/SecurityCloudGame.scss

```diff
@@ -37,20 +37,29 @@
         background-color: rgba(0, 0, 0, .1);
     }
 
-    .box-ui-security-cloud-game-board {
+    .bdl-SecurityCloudGame-board {
         position: relative;
         display: flex;
         flex: 1;
     }
 }
 
-.drag-cloud {
+.bdl-SecurityCloudGame-liveText {
+    position: absolute;
+    left: -9999px;
+}
+
+.bdl-DragCloud {
     position: absolute;
     cursor: move;
     fill: $white;
 }
 
-.drop-cloud {
+.bdl-DragCloud.is-moving {
+    fill: $bdl-gray-20;
+}
```

Comment:
could you combine this with the block above?

```
.bdl-DragCloud {
    position: absolute;
    cursor: move;
    fill: $white;

    &.is-moving {
        fill: $bdl-gray-20;
    }
}
```

Reply:
Done